src/Carnac/Views/ShellView.xaml

```diff
@@ -6,11 +6,12 @@
                       xmlns:viewModels="clr-namespace:Carnac.ViewModels"
                       xmlns:views="clr-namespace:Carnac.Views"
                       xmlns:xctk="http://schemas.xceed.com/wpf/xaml/toolkit" x:Class="Carnac.Views.ShellView"
-                      Width="610" Height="420" Icon="../icon.ico" ShowTitleBar="False"
+                      Width="610" Height="420" Icon="../icon.ico" 
                       Foreground="{DynamicResource BlackBrush}"
                       d:DataContext="{d:DesignInstance viewModels:ShellViewModel}" mc:Ignorable="d"
-                      ShowMinButton="False" ShowMaxRestoreButton="False"
-                      SaveWindowPosition="True">
+                      >
+    <!--ShowTitleBar="False" ShowMinButton="False" ShowMaxRestoreButton="False"
```

Comment:
Fixed in the next PR. For some reason the preferences window would not show up for me, I changed it to a normal window. These are MahApps.Metro properies, when I fixed it to use the MetroWindow again in the UI related PR I put these back.